### PR TITLE
--CRM-16188, added function to check the total amount and sum of line…

### PR DESCRIPTION
**Pull request by pradpnayak**
… total

### CODE REVIEW

**Comment by pradpnayak** [comment]
Exception will be thrown in api

**Comment by pradpnayak** [comment]
test failure related?

**Comment by pradpnayak** [comment]
jenkins, retest this please

File: tests/phpunit/CRM/Contribute/BAO/ContributionTest.php [hunk 1]
```diff
@@ -783,4 +783,52 @@ protected function addParticipantWithContribution() {
     return array($lineItems, $contributions);
   }
 
+  /**
+   * checkLineItems() check if total amount matches the sum of line total
```
Comment (eileenmcnaughton) [inline tests/phpunit/CRM/Contribute/BAO/ContributionTest.php]: Trivial point - the full comment standard (which we are encouraging but not enforcing) has a capital letter at the start & a full stop at the end

**Pull request merged**